Misc/NEWS.d/next/Library/2018-10-17-02-15-23.bpo-33947.SRuq3T.rst

```diff
@@ -0,0 +1 @@
+Fix bpo-33947: Dataclasses can raise RecursionError in __repr__ 
```

Comment:
The blurb should not reference the issue number. Please me this say "dataclasses no longer raise RecursionError in `__repr__`" (but without the backticks, which I added just to get this to display correctly in this comment).